cmake/Toolchain-Android.cmake

```diff
@@ -4,3 +4,6 @@ set(CMAKE_SYSTEM_NAME Android)
 
 # Use the clang tool set because it's support for C++11 is superior
 set(CMAKE_GENERATOR_TOOLSET DefaultClang)
+
+# Make sure we use all our processors when building
+set(CMAKE_ANDROID_PROCESS_MAX $ENV{NUMBER_OF_PROCESSORS})
```

Comment:
This doesn't look valid. Use https://github.com/Kitware/CMake/blob/master/Modules/ProcessorCount.cmake instead

Reply:
If you look at the link you provided, on Windows they use the exact same environment variable. At the moment I only test the android pipeline on Windows through visual studio. I haven't tested on linux/osx but you are right that it would be safer to use something cross platform.
I'll make a note and fix it in the next PR if it works.